src/views/event/components/EventCard.tsx

```diff
@@ -37,15 +37,14 @@ const EventCard: FC<EventCardProps> = ({ data, daftarEventHandler }) => {
   return (
     <Stack
       maxWidth="100%"
-      height="653px"
       bgcolor="neutral900.main"
       sx={{ borderRadius: '6px' }}
     >
       <img
         src={baseUrl.concat(data.image)}
         width="100%"
         height="157px"
-        style={{ objectFit: 'cover' }}
+        style={{ objectFit: 'cover', borderRadius: '6px 6px 0px 0px' }}
```

Comment:
gimana kalau 
```suggestion
        style={{ objectFit: 'cover' }}
```
dan stacknya dikasih overflow hidden

Reply:
ih keren bgt lu